Parser/asdl.py

```diff
@@ -339,14 +350,15 @@ def _parse_optional_attributes(self):
             return None
 
     def _parse_optional_field_quantifier(self):
-        is_seq, is_opt = False, False
-        if self.cur_token.kind == TokenKind.Asterisk:
-            is_seq = True
-            self._advance()
-        elif self.cur_token.kind == TokenKind.Question:
-            is_opt = True
+        quantifiers = []
+        while self.cur_token.kind in (TokenKind.Asterisk, TokenKind.Question):
```

Comment:
Shouldn't we check that we don't have `**` for instance or `??`?

Reply:
Maybe. But both ?? and ** are legitimate types, ** perhaps more legitimate than ?? is.

Reply:
Ah yes, actually the current parser doesn't care. I thought that it would have warned the user. And should `T**` actually mean a list of list of T? is the obtained C code designed to handle this kind of grammar?

I have no strong opinion now on whether we should warn `??`.

Reply:
Yeah I think this is fine as is